projects/1.16/assets/pneumaticcraft-repressurized/pneumaticcraft/patchouli_books/book/zh_cn/entries/armor/overview.json

```
@@ -19,49 +19,49 @@
+      "text": "穿戴盔甲时，盔甲将耗费数秒的时间$(thing)启动/$。可以按下 $(thing)$(k:pneumaticcraft.armor.options)/$ 键打开 GUI，在此可以配置安装的升级并调节 HUD。安装的升级越多，GUI 中出现选项的选项也越多。$(p)值得注意的是，所有升级都可以在 GUI 中通过复选框控制开关，甚至可以将开关操作与键位绑定，以便在未打开 GUI 时开关特定功能。"
     },
     {
       "type": "spotlight",
       "item": "minecraft:anvil",
       "title": "修复装甲",
-      "text": "有几种办法让这些珍贵的盔甲保持良好状态：$(li)可以在$(thing)铁砧/$里用压缩铁锭修复盔甲$(li)安装$(thing)物品寿命升级/$，以让装甲自动修复$(li)向每件盔甲附魔$(thing)经验修补/$"
+      "text": "有几种办法让这些昂贵的盔甲保持良好的状态：$(li)可以在$(thing)铁砧/$里用压缩铁锭修复盔甲$(li)给每件盔甲安装$(thing)物品寿命升级/$，以让装甲自动修复$(li)给每件盔甲附魔上$(thing)经验修补/$"
     },
     {
       "type": "text",
-      "title": "染色",
-      "text": "每一件气动盔甲都可以随时染色；其各自拥有独立的$(thing)主色/$和$(thing)次色/$设置；头盔的$(thing)目镜/$也可以独立染色。$(p)要调整色彩，在盔甲主界面中转至$(bold)色彩……/$界面。这一点不需要升级：这一功能是自带的。"
+      "title": "调色",
+      "text": "每一件气动盔甲都可以随时调色；其各自拥有独立的$(thing)主色/$和$(thing)次色/$设置；头盔的$(thing)目镜/$也可以独立调色。$(p)要调整色彩，在盔甲主 GUI 中转至$(bold)色彩……/$界面。不需要加装升级，这一功能是自带的。"
     },
     {
       "type": "spotlight",
       "item": "pneumaticcraft:speed_upgrade",
-      "text": "$(l:base_concepts/upgrades#speed)速度升级/$降低每件盔甲的启动时间。此外，安装在$(l:armor/pneumatic_helmet)气动头盔/$时，降低其执行$(l:base_concepts/upgrades#entity_tracker)实体追踪/$和$(l:base_concepts/upgrades#block_tracker)方块追踪/$任务的锁定时间；安装在$(l:armor/pneumatic_leggings)气动护腿/$时，提升疾跑的速率，但要$(l:base_concepts/pressure)消耗空气/$。"
+      "text": "$(l:base_concepts/upgrades#speed)速度升级/$降低每件盔甲的启动时间。此外，安装在$(l:armor/pneumatic_helmet)气动头盔/$时，降低其执行$(l:base_concepts/upgrades#entity_tracker)实体追踪/$和$(l:base_concepts/upgrades#block_tracker)方块追踪/$任务的锁定时间；安装在$(l:armor/pneumatic_leggings)气动护腿/$时，提升疾跑的速率，但会$(l:base_concepts/pressure)消耗空气/$。"
     },
     {
       "type": "spotlight",
       "item": "pneumaticcraft:volume_upgrade",
-      "text": "$(l:base_concepts/upgrades#volume)容量升级/$提升每件盔甲的$(l:base_concepts/pressure)空气容量/$；这会使其充气时间更长，同时压力随使用的衰减速率降低。可安装的数目最大为 25 个；且安装数量较大时将会有边际递减效应。"
+      "text": "$(l:base_concepts/upgrades#volume)容量升级/$提升每件盔甲的$(l:base_concepts/pressure)空气容量/$；这会使其充气时间变长，同时使用时压力的衰减速率会降低。可安装的数目最大为25个；且安装数量较大时单个升级的收益会不断降低。"
     },
     {
       "type": "spotlight",
       "item": "pneumaticcraft:armor_upgrade",
-      "text": "$(l:base_concepts/upgrades#armor)护甲升级/$提升每件盔甲的防御点数和盔甲韧性。安装2个升级后，每件盔甲的防护能力等价于对应的$(thing)钻石盔甲/$。安装最大4个升级后，这一防护能力就优于$(thing)钻石盔甲/$。"
+      "text": "$(l:base_concepts/upgrades#armor)护甲升级/$提升每件盔甲的护甲值和盔甲韧性。安装2个升级后，每件盔甲的防护能力等价于对应的$(thing)钻石盔甲/$。安装最大4个升级后，此时防护能力就强于$(thing)钻石盔甲/$。"
     },
     {
       "type": "spotlight",
       "item": "pneumaticcraft:item_life_upgrade",
-      "text": "$(l:base_concepts/upgrades#item_life)物品寿命升级/$缓慢地修复每件盔甲，同时消耗$(l:base_concepts/pressure)空气/$。最多可安装5个升级，从而提供显著更快（但以空气计效率更低）的修复。"
+      "text": "$(l:base_concepts/upgrades#item_life)物品寿命升级/$缓慢地修复盔甲，同时消耗$(l:base_concepts/pressure)空气/$。最多可安装5个升级，可以提供快的多（但消耗空气的效率更差）的修复速率。"
```

感觉“效率”有时候可能与“速度”混淆？
毕竟`高“效”地完成任务`指的好像一般是速度

```suggestion
      "text": "$(l:base_concepts/upgrades#item_life)物品寿命升级/$缓慢地修复盔甲，同时消耗$(l:base_concepts/pressure)空气/$。最多可安装5个升级，可以提供更快（但空气利用率更低）的修复速率。"
```
不过好像也不是特别合适

---

这里就是类似有损超频，无损超频那样的意思。就是修复速度更快，但是单位消耗量更大。我觉得改了以后的更好。